content/blog/mapr-for-predictive-maintenance.md

```diff
@@ -0,0 +1,318 @@
+
+This project is intended to show how to build Predictive Maintenance applications on MapR. Predictive Maintenance applications place high demands on data streaming, time-series data storage, and machine learning. Therefore, this project focuses on data ingest with MapR Streams, time-series data storage with MapR-DB and OpenTSDB, and feature engineering with MapR-DB and Apache Spark.
+
+## Overview:
+
+Predictive Maintenance requires a cutting edge data platform in order to handle fast streams of  IoT data, with the processing required for on-the-fly feature engineering, and the flexibility required for data science and machine learning.
+
+## Ingesting Factory IoT Data
+
+Predictive Maintenance applications rely heavily on ingesting multiple data sources, each with its own format and throughput. MapR Streams can ingest data, regardless of format or speed, with standard Kafka and RESTful APIs.
+
+## Machine Learning on Factory IoT Data
+
+The "predictive" aspects of Predictive Maintenance applications are usually realized through machine learning. Feature engineering is often considered the most important aspect of machine learning (as opposed to neural network design, for example). Feature engineering places high demands on the data layer because of the amount of data that IoT data streams generate. The tendency for failures to occur infrequently and without warning means vast amounts of raw time-series data must be stored. Not only must it be stored, but it must also be possible to retroactively update the lagging features necessary in order to label failures for the purposes of supervised machine learning. MapR-DB and Spark can work together to provide the capabiltieis required to put machine learning into practice for predictive maintanence.
+
+In summary:
+
+* MapR Streams provide a convenient way to ingest IoT data because it is scalable and provides convenient interfaces.
+* The integration of MapR DB with Spark provides a convenient way to label lagging features needed for predicting failures via supervised Machine Learning.
+* Drill provides a convenient way to load ML data sets into Tensorflow for unsupervised and supervised machine learning
+
+## Implementation Summary
+
+There are two objectives relating to predictive maintenance implemented in this project. The first objective is to visualize time-series data in an interactive real-time dashboard in Grafana. The second objective is to make raw data streams and derived features available to machine learning frameworks, such as Tensorflow, in order to develop algorithms for anomaly detection and predictive maintenance. These two objects are realized using two seperate data flows:
```

Comment:
You didn't initial cap "predictive maintenance" here. In looking up "predictive maintenance", I found that, as it relates to IoT, it tends to not be capitalized. Earlier, you were consistent in using initial caps, so I thought perhaps that, as an application title, it might be appropriate. I would be inclined to not use initial caps, but if you are going to use them as a proper noun for this subject, you'll need to be consistent throughout, including in the paragraph above.

Reply:
should be resolved in the body copy instances